### ISSUE TITLE
Swag for SfN

### ISSUE THREAD

**Issue by bendichter** [dandi/sfn2023#5]
- [ ] :+1: badge ribbons (e.g. from https://www.pcnametag.com/badge-ribbons/custom-ribbons.html) (@TheChymera)
   - "DANDI Contributor" -- @TheChymera has cute idea about different number of hats to signal amount of contribution
   - "My data is handy and in DANDI"
   - "I standardize my data" -- for anyone who standardizes and can show (eg BIDS on OpenNeuro)
   - "I Love Open Data on DANDI", "DANDI Data Gourmet" (for consumers of data)
   - "Proud DANDIset Owner"
   - "DANDI/NWB Funder" (any US taxpayer is)
   - "A Dandy DANDIset owner"
   - "NWB Wizard"
   - "Not a Neuroscientist"
   - any other idea? 
- [ ] :-1: tshirts (@asmacdo)
- [ ] :heart:  tophats (@TheChymera) ...
- [ ] :eyes: ...
- [ ] 

More ideas, and who is willing to take care about it if decided that it is worthwhile.
emojis are for voting -- vote for what you think is worthwhile.

**Comment by TheChymera** [comment]
Top hats:

* https://www.amazon.com/Gemvie-Stage-Adults-Costume-56-58cm/dp/B076BMNQYG/ref=sr_1_35 -- we are ordering one of this to try!
* https://www.amazon.com/Amscan-Deluxe-Black-Costume-Accessory/dp/B01IPRU266/ref=sr_1_1

**Comment by TheChymera** [comment]
For the badges I think it would be nice to play off of the dandy DANDI look, maybe with some pseudo-honorific titles:

* Astute Open Data enthusiast
* Suave DANDIset user
* Dandy DANDIset owner
* Sophisticated DANDI contributor
* Sophisticated NWB contributor

I think this would make it memorable and initiate discussions.
The latter words are either potentially unrecognized acronyms or just generic badge terms, I think we would get better outreach if they react to the first word with surprise. 

We shouldn't have too many versions, minimum order size per unique text blurb is 100, costing ~40 USD.

**Comment by asmacdo** [comment]
![dandiset-owner](https://github.com/dandi/sfn2023/assets/1028657/75f89c6b-a286-4c07-ab3b-de3a9fde5d6c)
![dandi-user](https://github.com/dandi/sfn2023/assets/1028657/48c953c0-fba8-4bf5-83cb-052a087a3555)
![dandi-contributor](https://github.com/dandi/sfn2023/assets/1028657/233c97eb-f5df-410c-aef3-a8a680e108d7)

**Comment by TheChymera** [comment]
@asmacdo sounds great!

**Comment by yarikoptic** [comment]
please add 100 of `I standardize my data`, `NWB Wizard`

**Comment by asmacdo** [comment]
![test-swoosh](https://github.com/dandi/sfn2023/assets/1028657/94c1f2b0-e64d-4e2f-8d9f-363863a5594e)

**Comment by TheChymera** [comment]
@asmacdo you said the NWB logo doesn't look that good in monochrome, have a look at the following, I think it might work:

![nwb-logo_monochrome](https://github.com/dandi/sfn2023/assets/950524/e86a7ede-4d28-49bb-9f3a-d72af8a9ac22)

SVG here → https://ppb.chymera.eu/d569ca.svg


@bendichter what do you say, does it represent you well enough, or would you rather go logo-free?

**Comment by asmacdo** [comment]
Next draft

![dandi-contributor](https://github.com/dandi/sfn2023/assets/1028657/198e4f80-e71b-492f-a3e0-a6a921122446)
![dandi-set-user](https://github.com/dandi/sfn2023/assets/1028657/efc12b9f-3ecd-47e0-9610-e27a84d47709)
![dandiset-owner](https://github.com/dandi/sfn2023/assets/1028657/40e1e1e2-c4f7-4979-8e38-c2b7f59fecfb)
![i-standardized-data](https://github.com/dandi/sfn2023/assets/1028657/0aba92f7-f3bc-4a41-9bd3-f317ff6c612d)
![nwb-wizard](https://github.com/dandi/sfn2023/assets/1028657/da2ff9ad-b7a5-4bd0-bbde-f04cd360ec46)

**Comment by oruebel** [comment]
Those ribbons look nice! 

- For "NWB Wizard" I would  add a space before the text to have a bit more space between the logo and text
- For  the "I Standardize My Data" it would be cool to have the NWB Logo too

**Comment by bendichter** [comment]
I agree with @oruebel . I think this looks good, I would just suggest to add a space

**Comment by sneakers-the-rat** [comment]
> ![test-swoosh](https://user-images.githubusercontent.com/1028657/278085395-94c1f2b0-e64d-4e2f-8d9f-363863a5594e.jpg)

so i love the actual ones but also srsly print some of these too

**Comment by asmacdo** [comment]
I've already put the order in, but i would also wear a TEST 😂
